build.sh

```diff
@@ -142,11 +142,16 @@ fi
 
 cp Dockerfile.cross Dockerfile.build
 
-sed -i "s|__TAG__|${tag}|g" Dockerfile.build
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  sed -i "" "s|__TAG__|${tag}|g" Dockerfile.build
+  sed -i "" "s|__DEPENDENCIES__|${dependencies}|g" Dockerfile.build
+else
+  sed -i "s|__TAG__|${tag}|g" Dockerfile.build
+  sed -i "s|__DEPENDENCIES__|${dependencies}|g" Dockerfile.build
+fi
```

Comment:
isn't this useless? you are doing the same thing in both if and else

Reply:
nope on osx sed requires passing an empty string as the first parameter. I added this to make the script runnable on osx. However, I can remove it of you don't want to make it runnable on osx machines :D

Reply:
Oh sorry completely missed that empty string :open_mouth:

Reply:
Leave it there no problem :)